src/test/java/org/grizz/web/api/SampleData.java

```diff
@@ -0,0 +1,37 @@
+package org.grizz.web.api;
+
+import org.grizz.OgameCloneApplication;
+import org.grizz.model.Building;
+import org.grizz.model.Planet;
+import org.grizz.model.repos.PlanetRepository;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = OgameCloneApplication.class)
+@WebAppConfiguration
+public class SampleData {
+
+    @Autowired
+    PlanetRepository planetRepository;
+
+    // @Ignore
```

Comment:
CRITICAL: Move this test to another package. It has nothing in common with web.api!!